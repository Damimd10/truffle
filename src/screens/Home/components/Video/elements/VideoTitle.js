import styled from 'styled-components';

const VideoTitle = styled.span`
  color: #2a3438;
  font-weight: bold;
  font-size: 1.3rem;
`;

export default VideoTitle;
