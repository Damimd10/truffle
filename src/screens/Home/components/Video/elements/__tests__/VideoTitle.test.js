import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VideoTitle from '../VideoTitle';

describe('Button Style Element', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<VideoTitle />);
  });

  it('should create a style structure for the component', () =>
    expect(toJson(wrapper)).toMatchSnapshot());
});
