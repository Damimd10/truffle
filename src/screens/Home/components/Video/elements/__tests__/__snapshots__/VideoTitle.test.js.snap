// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Style Element should create a style structure for the component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  color: #2a3438;
  font-weight: bold;
  font-size: 1.3rem;
",
        ],
      },
      "displayName": "VideoTitle",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "span",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
